scripts/sonic_sku_create.py

```diff
@@ -0,0 +1,853 @@
+  -c [{new_sku_only,l2_mode_only}], --cmd [{new_sku_only,l2_mode_only}]
+                        Choose action to preform (Generate a new SKU, Configure L2 mode, Both
+  -k HWSKU, --hwsku HWSKU
+                        SKU name to be used when creating a new SKU or for  L2 configuration mode
+  -p, --print           Print port_config.ini without creating a new SKU
+  -vv, --verbose        Verbose output
+
+
+"""
+
+from __future__ import print_function
+
+import argparse
+import json
+import os
+import re
+import subprocess
+import swsssdk
+import traceback
+import sys
+import shutil
+import copy
+from collections import OrderedDict
+
+from tabulate import tabulate
+from lxml import etree as ET
+from lxml.etree import QName
+
+minigraph_ns = "Microsoft.Search.Autopilot.Evolution"
+minigraph_ns1 = "http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution"
+INTERFACE_KEY = "Ethernet"
+
+test_envir = None
+try:
+	#test_envir = os.environ.get('UTILITIES_UNIT_TESTING')
+	test_envir = os.environ.get('USER')
+except KeyError:
+	print ("Not Runing on unit test envir", file=sys.stderr)
+
+### port_config.ini header
+PORTCONFIG_HEADER = ["# name", "lanes", "alias", "index", "speed"]
+platform_4 = ['x86_64-mlnx_lssn2700-r0','x86_64-mlnx_msn2010-r0','x86_64-mlnx_msn2100-r0','x86_64-mlnx_msn2410-r0','x86_64-mlnx_msn2700-r0','x86_64-mlnx_msn2740-r0','x86_64-mlnx_msn3700c-r0','x86_64-mlnx_msn3700-r0','x86_64-mlnx_msn3800-r0']
+platform_8 = ['x86_64-mlnx_msn4600c-r0','x86_64-mlnx_msn4700-r0']
+
+bko_dict_4 = {
+        "1x100": { "lanes":4, "speed":100000, "step":4, "bko":0, "name": "etp" },
+        "1x40":  { "lanes":4, "speed":40000,  "step":4, "bko":0, "name": "etp" },
+        "1x50":  { "lanes":4, "speed":50000,  "step":4, "bko":0, "name": "etp" },
+        "1x25":  { "lanes":4, "speed":25000,  "step":4, "bko":0, "name": "etp" },
+        "1x10":  { "lanes":4, "speed":10000,  "step":4, "bko":0, "name": "etp" },
+        "1x1":   { "lanes":4, "speed":1000,   "step":4, "bko":0, "name": "etp" },
+        "4x10":  { "lanes":4, "speed":10000,  "step":1, "bko":1, "name": "etp" },
+        "4x25":  { "lanes":4, "speed":25000,  "step":1, "bko":1, "name": "etp" },
+        "2x50":  { "lanes":4, "speed":25000,  "step":2, "bko":1, "name": "etp" },
+}
+
+bko_dict_8 = {
+        "1x400": { "lanes":8, "speed":400000, "step":8, "bko":0, "name": "etp" },
+        "2x200": { "lanes":8, "speed":200000, "step":4, "bko":1, "name": "etp" },
+        "2x100": { "lanes":8, "speed":100000, "step":4, "bko":1, "name": "etp" },
+        "4x100": { "lanes":8, "speed":100000, "step":2, "bko":1, "name": "etp" },
+        "4x50":  { "lanes":8, "speed":50000,  "step":2, "bko":1, "name": "etp" },
+        "4x25":  { "lanes":4, "speed":25000,  "step":1, "bko":1, "name": "etp" },
+        "4x10":  { "lanes":4, "speed":10000,  "step":1, "bko":1, "name": "etp" },
+}
+
+bko_dict = bko_dict_4
+
+class SkuCreate(object):
+        """
+	Process aclstat
```

Comment:
@madhanmellanox what is `aclstat`?

Reply:
modified the comments. Addressed in next commit.